skyportal/handlers/api/sysinfo.py

```diff
@@ -26,14 +32,61 @@ def get(self):
                         properties:
                           invitationsEnabled:
                             type: boolean
-                          description: |
-                            Boolean indicating whether new user invitation pipeline
-                            is enabled in current deployment.
+                            description: |
+                              Boolean indicating whether new user invitation pipeline
+                              is enabled in current deployment.
+                          gitlog:
+                              type: array
+                              description: Recent git commit lines
+                          cosmology:
+                              type: string
+                              description: Details of the cosmology used here
+                          cosmoref:
+                              type: string
+                              description: Reference for the cosmology used.
         """
+        # if another build system has written a gitlog file, use it
+        loginfo = None
+        if os.path.exists(gitlog_file):
+            with open(gitlog_file, "r") as spgl:
+                loginfo = spgl.read()
+        if not loginfo:
+            p = subprocess.run(
+                [
+                    "git",
+                    "--git-dir=.git",
+                    "log",
+                    "--pretty=format:'%C(cyan)[%ci]%Creset %s %C(auto)%h'",
+                ],
+                capture_output=True,
+                universal_newlines=True,
+            )
+            loginfo = p.stdout
+
+        raw_gitlog = loginfo.splitlines()[:default_log_lines]
+        gitlog = []
```

Comment:
Looking at this block, `loginfo` starts as `None` and then if the external logfile exists it reads that and if not it runs `git` (which expects a `.git` directory to be there) By the time it gets to L66 then it expects `loginfo` to have been populated as a string. But if there is no `.git` and no external log then it will still be `None` and L66 will throw an exception.

Reply:
I think it would be something like

```
AttributeError: 'NoneType' object has no attribute 'splitlines'
```

Reply:
Fixed. Using `""` as the default. `"".splitlines` will produce `[]`, which will not error.